File: code/ex1.py
class Solution:
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        result = []
        i = 0
        j = 0
        
        for i in range(0, len(nums)):
            for j in range(i + 1, len(nums)):
                if nums[i] + nums[j] == target:
                    result.append(i)
                    result.append(j)


        return result
            

sol = Solution()

print(sol.twoSum([3,2,4], 6))

File: code/ex1.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        int n = nums.size();
        unordered_map<int, int> map;
        for (int i = 0; i < n; i++)    map[nums[i]] = i;
        for (int i = 0; i < n; i++) {
            int t = target - nums[i];
            if (map.count(t) > 0 && map[t] != i)    return {i, map[t]};
        }
        return {-1, -1};
    }
};